usr/sbin/registercloudguest

```diff
@@ -1,682 +1,19 @@
-#!/usr/bin/python3
+#!/bin/bash
+set -e
 
-# Copyright (c) 2024, SUSE LLC, All rights reserved.
-#
-# This library is free software; you can redistribute it and/or
-# modify it under the terms of the GNU Lesser General Public
-# License as published by the Free Software Foundation; either
-# version 3.0 of the License, or (at your option) any later version.
-# This library is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-# Lesser General Public License for more details.
-# You should have received a copy of the GNU Lesser General Public
-# License along with this library.
+# store pid of parent shell
+parent=$PPID
 
-"""This script obtains information from the configured region server in the
-   cloud environment and uses the information to register the guest with
-   the SMT server based on the information provided by the region server.
+registercloudguest.py "$@"
```

Comment:
should we be explicit on the path ?
`/usr/sbin/registercloudguest.py`

Reply:
Hmm, it's unlikely that we change the location, ok let's make it sticky. done